catalogue_pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAgents.scala

```diff
@@ -27,11 +27,18 @@ trait SierraAgents {
       val prefixString =
         if (prefixes.isEmpty) None else Some(prefixes.mkString(" "))
 
-      Person(
-        label = label,
-        prefix = prefixString,
-        numeration = numeration
-      )
+      if (Set("100", "700").contains(marcTag))
```

Comment:
I would rather pass in a parameter `normalisePerson: Boolean` than the MARC tag, that defaults to false – otherwise we have to know about what a particular MARC tag means in multiple places.